src/main/kotlin/uk/gov/dluhc/printapi/client/ElectoralRegistrationOfficeManagementApiClient.kt

```diff
@@ -21,43 +22,56 @@ class ElectoralRegistrationOfficeManagementApiClient(
     private val eroMapper: EroManagementApiEroDtoMapper
 ) {
 
+    /**
+     * Calls the `ero-management-api` to return a [ElectoralRegistrationOfficeResponse] for the specified eroId.
+     *
+     * @param eroId the ID of the ERO to return
+     * @return a [ElectoralRegistrationOfficeResponse] for the ERO
+     * @throws [ElectoralRegistrationOfficeManagementApiException] concrete implementation if the API returns an error
+     */
+    fun getElectoralRegistrationOffice(eroId: String): ElectoralRegistrationOfficeResponse =
+        eroManagementWebClient
```

Comment:
@Neil-Massey , I think it will be better if we don't expose the external `ElectoralRegistrationOfficeResponse` to the service layer and return a dto instead here like we did with the method below.

Reply:
This is a copy of approach from VCA:
https://github.com/cabinetoffice/eip-ero-voter-card-applications-api/blob/main/src/main/kotlin/uk/gov/dluhc/votercardapplicationsapi/client/ElectoralRegistrationOfficeManagementApiClient.kt

Reply:
But will review suggestion on Monday as your approach may be better 😄 .

Reply:
Done - better to map to internal model 👍

Reply:
Thanks, @Neil-Massey.